docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,225 @@
+```
+To run the code you just cloned, install all the necesasry packages. 
+```bash
+npm install
+```
+
+:::caution
+Prior to running these scripts, ensure you replace all instances of `keypom-docs-demo.testnet` in the script with the credentials of your account found in your `~/.near-credentials` folder
+:::
+
+From there, you are able to run this Simple Drop script that was made in this tutorial using the following command:
+``` bash
+npm run simple-keypom
+```
+:::note
+The SDK script is being tested here; use `npm run simple-near` to test the NEAR-API-JS script instead.
+:::
+This should return a successful drop creation and console log a Public Key and Linkdrop: 
+```bash
+Public Keys and Linkdrops:  {
+  'ed25519:8nCdZ32nRUdZRXs4dSgjq2FiJnsgMSJYpxNJ4cg7Z1iz': 'https://wallet.testnet.near.org/linkdrop/v1-3.keypom.testnet/5gzHZeLFDyDJHeCVtACwQxbMj8BRTeuu1WVd7eSNhjtc89uZjCoxXW3vh8R1RwEK44w3rb7sBAqMg5hR1MCHsC6e'
+}
+```
+To see the full console log from this drop creation, see the expandable section below.
+
+<details>
+<summary>Console Log of Test</summary>
+<p>
+
+``` bash
+yarn run v1.22.19
+warning ../../../package.json: No license field
+$ node docs-examples/keypom-js-sdk/simple-example.js
+Receipts: AJ5so8mEDh9XEYNcB7vomTxUDV3BAMyLVA7QMguHuscA, 4YyHuHyMh1gZimE2Ep3Y4boasFj5cbXr9TmL2qkNkG6Z
+        Log [v1-3.keypom.testnet]: Current Block Timestamp: 1674847726484923818
+        Log [v1-3.keypom.testnet]: 20 calls with 100000000000000 attached GAS. Pow outcome: 1.8061103. Required Allowance: 18762630063718400000000
+        Log [v1-3.keypom.testnet]: Total required storage Yocto 11110000000000000000000
+        Log [v1-3.keypom.testnet]: Current balance: 1.2083208, 
+            Required Deposit: 1.0308726, 
+            total_required_storage: 0.01111,
+            Drop Fee: 0, 
+            Key Fee: 0 Total Key Fee: 0,
+            allowance: 0.0187626 total allowance: 0.0187626,
+            access key storage: 0.001 total access key storage: 0.001,
+            deposits less none FCs: 1 total deposits: 1 lazy registration: false,
+            deposits for FCs: 0 total deposits for FCs: 0,
+            uses per key: 1
+            None FCs: 0,
+            length: 1
+            GAS to attach: 100000000000000
+        Log [v1-3.keypom.testnet]: New user balance 0.1774482
+        Log [v1-3.keypom.testnet]: Fees collected 0
+Public Keys and Linkdrops:  {
+  'ed25519:8nCdZ32nRUdZRXs4dSgjq2FiJnsgMSJYpxNJ4cg7Z1iz': 'https://wallet.testnet.near.org/linkdrop/v1-3.keypom.testnet/5gzHZeLFDyDJHeCVtACwQxbMj8BRTeuu1WVd7eSNhjtc89uZjCoxXW3vh8R1RwEK44w3rb7sBAqMg5hR1MCHsC6e'
+}
+```
+
+</p>
+</details>
+
+### Claiming and Explorer Transactions
+Once you have the link, you are able to claim the linkdrop you've just created. The output link will take you to the following NEAR Wallet page, where you will have the choice to call claim to an existing account or create a new one. 
```

Comment:
```suggestion
Once you have the link, you are able to claim the linkdrop you've just created. Once clicked, it will take you to the following NEAR Wallet page, where you will have the choice to claim with an existing account or create a new one. 
```